src/shared/components/structureViewer/StructureViewer.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {observer} from "mobx-react";
+import {
+    IStructureVisualizerProps, IResidueSpec
+} from "./StructureVisualizer";
+import StructureVisualizer3D from "./StructureVisualizer3D";
+
+export interface IStructureViewerProps extends IStructureVisualizerProps {
+    pdbId: string;
+    chainId: string;
+    residues?: IResidueSpec[];
+}
+
+@observer
+export default class StructureViewer extends React.Component<IStructureViewerProps, {}>
+{
+    private _3dMolDiv: HTMLDivElement|undefined;
+    private _pdbId: string;
+    private wrapper: StructureVisualizer3D;
+
+    public constructor() {
+        super();
+
+        this.divHandler = this.divHandler.bind(this);
+    }
+
+    public render()
+    {
+        return (
+            <div
+                ref={this.divHandler}
+                style={{height: "300px"}}
+            />
+        );
+    }
+
+    public componentDidMount() {
+        if (this._3dMolDiv) {
```

Comment:
Not really. Variable names cannot start with a number, so it was not possible to name it `3dMolDiv`, so I decided to start with an underscore. This may not be the best approach...